test/common/router/config_impl_test.cc

```diff
@@ -6692,62 +6693,62 @@ TEST_F(RoutePropertyTest, TestVHostCorsConfig) {
   EXPECT_EQ(cors_policy->exposeHeaders(), "test-expose-headers");
   EXPECT_EQ(cors_policy->maxAge(), "test-max-age");
   EXPECT_EQ(cors_policy->allowCredentials(), true);
+  EXPECT_EQ(cors_policy->allowPrivateNetworkAccess(), true);
 }
 
-TEST_F(RoutePropertyTest, TestRouteCorsConfig) {
+TEST_F(RoutePropertyTest, TestVHostCorsConfigDisablePNA) {
   const std::string yaml = R"EOF(
 virtual_hosts:
   - name: "default"
     domains: ["*"]
+    cors:
+      allow_private_network_access: false
```

Comment:
Could we keep the original test case here and just add a new simple case for `DisablePNA`?